vmware_exporter/vmware_exporter.py

```diff
@@ -2027,6 +2100,8 @@ def main(argv=None):
                         default=9272, help="HTTP port to expose metrics")
     parser.add_argument('-l', '--loglevel', dest='loglevel',
                         default="INFO", help="Set application loglevel INFO, DEBUG")
+    parser.add_argument('--allow-url-target', action='store_true',
```

Comment:
This can be done via an env variable, not sure it needs to be a arg as well.

Reply:
This change fixes a security issue where by default you would be able to use the username/password from any section to login to any vcenter accessable from the collector.